src/Raven.Server/Documents/Transformers/CurrentTransformationScope.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Raven.Server.Documents.Indexes.Static;
+
+namespace Raven.Server.Documents.Transformers
+{
+    public class CurrentTransformationScope
+    {
+        private readonly DocumentDatabase _documentDatabase;
+
+        public readonly IndexingFunc TransformResults;
+
+        [ThreadStatic]
+        public static CurrentTransformationScope Current;
```

Comment:
Let us change this later, I don't _think_ we'll call out in the middle. Just need to be aware of this.